pkg/manifests/config.go

```diff
@@ -89,6 +89,7 @@ type PrometheusK8sConfig struct {
 	Hostport            string                    `json:"hostport"`
 	RemoteWrite         []monv1.RemoteWriteSpec   `json:"remoteWrite"`
 	TelemetryMatches    []string                  `json:"-"`
+	Replicas            int32                     `json:"replicas"`
```

Comment:
Since it only applies to the user-workload Prometheus, shouldn't we maintain 2 config structs to avoid confusion? I see other fields (e.g. `TelemetryMatches` that only apply to the infra Prometheus.

Reply:
sounds good :+1: I will split the struct.

Reply:
fixed